mesh.py

```diff
@@ -44,24 +62,44 @@ def flux_add(self, cell, distance):
     def flux_clear(self):
         self._flux[:][:][:] = 0
 
-    def display_flux(self, index):
-        plotter.plot_heat_map(self._flux, index)
-    
+    def display_flux(self):
```

Comment:
We've gotten a bit lazy with this, and it's probably my fault. But all functions should come with a description like:
```
'''
@brief prints the flux stored in every cell to the screen
'''
```